frontend/src/components/forms/order/Order.tsx

```diff
@@ -45,7 +45,7 @@ function Order(props: Props) {
     const selectSupplier = (e: ChangeEvent<HTMLInputElement>) => {
         const supplier = suppliers.find(s => s.id === e.target.value);
```

Comment:
Abkürzungen s und e machen den Code zumindest für mich schwer nachvollziehbar

Reply:
das s meinetwegen. aber das e ist in den Parametern ja eindeutig typisiert. Soviel kann man von einem Reviewer erwarten, finde ich ;-)